solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,117 @@
+package ru.sberbank.school.task06;
+
+import ru.sberbank.school.task06.Player;
+
+import java.time.Period;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
+ *
+ *
+ * Пример(без дженериков):
+ *
+ * CountMap map = new ... ();
+ *
+ * map.add("Some word");
+ *
+ * map.add("Some word");
+ *
+ *
+ *
+ * ==> map{"Some word" : 2}
+ */
+public class CountMapImpl implements CountMap<Player> {
+
+    private HashMap<Player, Integer> playersMap = new HashMap<>();
+    /**
+     * Добавление элепмента в контейнер.
+     *
+     * @param p элемент для добавления
+     */
+    public void add(Player p) {
+        playersMap.put(p, playersMap.getOrDefault(p, 0)+1);
```

Comment:
WhitespaceAround: &#39;+&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1707418868>*